src/errors/ValueSetFilterOperatorError.ts

```diff
@@ -0,0 +1,5 @@
+export class ValueSetFilterOperatorError extends Error {
+  constructor(public value: string) {
+    super(`Operator "${value}" not available in value set filter.`);
```

Comment:
Thanks for reminding me about the `Annotated` interface, I had forgotten about it and was wondering "huh, should I list things that are allowed when I show the error?" Yes, I should, and this is how.

Reply:
I took another look at spec's definition for `property`, and it might actually be correct to allow any string (and not throw an error when it isn't one of the four specified). All it says on http://hl7.org/fhir/valueset-definitions.html#ValueSet.compose.include.filter.property is that it needs a code which "identifies a property or a filter defined in the code system." The four enum values are the four properties of a `code`, but since system-specific filters can be defined, "a filter defined in the code system" could basically be anything. I'll keep the current behavior for now, but every time I reread that part of the spec, it seems more vague.

Reply:
I think you are right that we may want to be more permissive here. For example, for this ValueSet from mCODE here: http://standardhealthrecord.org/guides/mcode-r4/ValueSet-onco-core-CancerBodyLocationVS.json.html, wouldn't we want to say `* codes from SCT where concept is-a 123037004`? But I think that wouldn't be allowed. If that is correct, I would vote that we be more permissive here. Do you think it is too permissive to just allow any single word property here? You're right, the spec is a little frustratingly vague though.